doc/lsst.ip.diffim/tasks/lsst.ip.diffim.DecorrelateALKernalSpatialTask.rst

```diff
@@ -0,0 +1,28 @@
+.. lsst-task-topic:: lsst.ip.diffim.DecorrelateALKernelSpatialTask
+
+##########################
+DecorrelateALKernelSpatialTask
+##########################
+
+Pipe-task that removes the neighboring-pixel covariance in an
+image difference that are added when the template image is
+convolved with the Alard-Lupton PSF matching kernel.
+
+.. _lsst.ip.diffim.DecorrelateALKernelSpatialTask-description:
+
+Description
+==================
```

Comment:
Section headers need the underlines to match the length of the header text.

https://developer.lsst.io/restructuredtext/style.html#sections